data/s3-synapse-sync-bucket-vars.yaml

```diff
@@ -0,0 +1,5 @@
+s3-synapse-sync-bucket-a:
+  SynapseProjectId: "syn11111"
```

Comment:
I have this synapse project to test with: syn22335495

Reply:
updated.